Proving methods in AError
Starting proof of AError.AError() with prover !!!!
Feasibility check #1 - end of preconditions : OK
Feasibility check #2 - at program exit : OK
Completed proof of AError.AError() with prover !!!! - no warnings
Starting proof of AError.m() with prover !!!!
test/gitbug535/AError.java:6: warning: The prover cannot establish an assertion (Assert) in method m
   throw new AssertionError();
   ^
Completed proof of AError.m() with prover !!!! - with warnings
Starting proof of AError.mm() with prover !!!!
test/gitbug535/AError.java:12: warning: The prover cannot establish an assertion (Assert) in method mm
   throw new AssertionError();
   ^
Completed proof of AError.mm() with prover !!!! - with warnings
Starting proof of AError.mmm() with prover !!!!
test/gitbug535/AError.java:19: warning: The prover cannot establish an assertion (Assert) in method mmm
   throw new AssertionError();
   ^
Completed proof of AError.mmm() with prover !!!! - with warnings
Completed proving methods in AError
3 warnings
